docs/source/pillars/information_governance.md

```diff
@@ -177,6 +177,9 @@ _The ability of the TRE organisation to ensure that people with access to data a
 * - You have a robust method for identifying accredited members of your TRE organisation, prior to their accessing of sensitive data
   - This may include multi-factor authentication (MFA), ID checks or email/phone verification
   - Mandatory
+* - You have a clear method of communicating the level of security controls present in the TRE for all roles within your TRE organisation
+  - This will be particularly important researchers working with the data, to understand what actions they are and are not permitted to take.
```

Comment:
```suggestion
  - People in every role should understand the security controls, the need for those controls, and how those controls impact on their responsibilities.
    For example, it should be clear what actions people are not permitted to take when controls are in place to prevent them.
```

Is this what we mean?

Reply:
I think users knowing which actions are and aren't allowed is part of user training. It isn't inherent to the idea of security tiers. As above, I'm not sure we need this point at all.

Reply:
Agreed.

There isn't an item in "Training management and delivery" which specifically says allowed/not allowed actions should be covered.
Do we want to add this there or not include it at all.

Reply:
I think we should add one (can be in this PR) that says something like "You must ensure that users are aware which actions are and are not permitted in your TRE // This is likely to depend on the security controls in place in your TRE // Mandatory"

Reply:
Would this user point not be included in a user terms of use point in information governance?